js/tic_tac_toe.js

```diff
@@ -1,2 +1,26 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function() {
+
+  var lastColor = 'red';
+
+  // gives me all elements with a class of box_cell
+  $('table tbody tr').on('click', 'td', function(event){
```

Comment:
I like the way you're using the event delegation here, but I'm not sure if its needed since we aren't creating new elements ever. You could probably just tie it straight to all the td's